content/courses/onchain-development/intro-to-anchor.md

```diff
@@ -117,19 +138,27 @@ instruction requires. When you use `Context`, you specify the concrete type of
 `Context<AddMovieReviewAccounts>`). Through this context argument the
 instruction can then access:
 
+Through this context argument the instruction can then access:
+
 - The accounts passed into the instruction (`ctx.accounts`)
 - The program ID (`ctx.program_id`) of the executing program
 - The remaining accounts (`ctx.remaining_accounts`). The `remaining_accounts` is
   a vector that contains all accounts that were passed into the instruction but
   are not declared in the `Accounts` struct.
 - The bumps for any PDA accounts in the `Accounts` struct (`ctx.bumps`)
+- The seeds for any PDA accounts in tha `Accounts` struct (`ctx.seeds`)
+
+> The design of Contexts can be different across different programs to serve
+> their purpose; and the name of the context could be anything (not limited to
+> Context) to better reflect it's usage. This example is to help understand how
+> contexts work in Anchor.
 
 ### Define instruction accounts
 
-The `Accounts` trait defines a data structure of validated accounts. Structs
-adopting this trait define the list of accounts required for a given
-instruction. These accounts are then exposed through an instruction's `Context`
-so that manual account iteration and deserialization is no longer necessary.
+The `Accounts` trait defines a structure of validated accounts. Structs that
+adopt this trait specify the accounts required for an instruction. These
+accounts are then accessible through an instruction's `Context`, eliminating the
+need for manual account handling and deserialization.
```

Comment:
The old version of this was needlessly complex. Let's make it way better. 

> One of the benefits of Solana compared to older blockchains is that transactions that involve different accounts don't stop each other from being processed. Alice can send USDC to Bob while Charles makes a transaction to Diane's lending protocol, and neither transaction will block the other. This means that the incoming instructions must include a list of the accounts that will be used in the instruction. 

> The `Accounts` trait is applied to a `struct` that specifies the accounts required for each instruction, as well as any additional requirements for those accounts - for example requiring a particular account to have signed the transaction. These accounts are then accessible through an instruction's `context` parameter.

Just a suggestion. Feel free to write your own!

Reply:
sure, I can try